usl_pipeline/cloud_functions/main.py

```diff
@@ -1011,6 +1019,10 @@ def _write_flood_chunk_metastore_entry(chunk_blob: storage.Blob) -> None:
         needs_scaling=True,
         x_index=x_index,
         y_index=y_index,
+        col_count=header.col_count,
+        row_count=header.row_count,
+        x_ll_corner=header.x_ll_corner,
+        y_ll_corner=header.y_ll_corner,
```

Comment:
Are these the values from the study area?
Would all the chunks have the same value for these?
I would either
- not replicate information from the study area in the chunks and have callers refer to the study area
- calculate the x & y ll_corner for this specific chunk, which is how I would expect folks would interpret this field (as being for the specific chunk rather than the study area)

Reply:
No, this is not a replica from study area metadata. These fields correspond to properties of sub-area of this specific chunk.